PeriodogramAnalysis/test_power_density_statistics.py

```diff
@@ -75,22 +81,28 @@ def test_spectrum_analysis_computation(analysis_class) -> None:
         assert range_psd_sum / total_psd_sum >= 0.9
 
         for i, freq in enumerate(freqs):
-            if all(abs(freq - exp_freq) > 1 for exp_freq in expected_freqs):
+            if all(abs(freq - exp_freq) > 0.5 for exp_freq in expected_freqs):
                 assert psd[i, channel] < 0.05 * spike_psd
 
 
 @pytest.mark.parametrize(
-    "analysis_class",
-    [SpectrumAnalysisWelch, SpectrumAnalysisPeriodogram],
+    "analysis_class, rate",
```

Comment:
fix this
<img width="825" alt="image" src="https://github.com/user-attachments/assets/eeaad328-835a-4eed-b77b-717368004a24" />

Reply:
Have fixed it.